src/app/components/RouterTabs/index.tsx

```diff
@@ -18,13 +18,21 @@ function getPathname(tab: { to: string }) {
 
 export function RouterTabs<Context>({ tabs, context }: RouterTabsProps<Context>) {
```

Comment:
Ideally add some tests, e.g. what tab is selected if tabs are
```tsx
tabs = [
  "/mainnet/sapphire/address/oasis1qry5d5zxs58w9deu5ara5zaamhkzqtg99s00ls90",
  "/mainnet/sapphire/address/oasis1qry5d5zxs58w9deu5ara5zaamhkzqtg99s00ls90/token-transfers#transfers",
  "/mainnet/sapphire/address/oasis1qry5d5zxs58w9deu5ara5zaamhkzqtg99s00ls90/tokens/erc-20#tokens",
  "/mainnet/sapphire/address/oasis1qry5d5zxs58w9deu5ara5zaamhkzqtg99s00ls90/tokens/erc-721#tokens",
  "/mainnet/sapphire/address/oasis1qry5d5zxs58w9deu5ara5zaamhkzqtg99s00ls90/code#code",
]
```

if location is:
/mainnet/sapphire/address/oasis1qry5d5zxs58w9deu5ara5zaamhkzqtg99s00ls90/tokens/erc-721
/mainnet/sapphire/address/oasis1qry5d5zxs58w9deu5ara5zaamhkzqtg99s00ls90/tokens/erc-721/token1#token1
/mainnet/sapphire/address/oasis1qry5d5zxs58w9deu5ara5zaamhkzqtg99s00ls90/other
/mainnet/sapphire/address/oasis1qry5d5zxs58w9deu5ara5zaamhkzqtg99s00ls90#transfers

Reply:
I will add them once I will wrap up other NFT UI work.